core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-filter.model.ts

```diff
@@ -23,24 +23,28 @@ export class ContributorAdminDashboardFilter {
   topicIds: string[];
   languageCode?: string;
   sort?: string | null;
-  lastActivity?: number;
+  numberOfDaysSinceLastActivityStartDate?: number;
+  numberOfDaysSinceLastActivityEndDate?: number;
 
   /**
    * @param {String} languageCode - Language Code to filter for.
    * @param {String[]} topicIds - keywords to filter for.
    * @param {string} sort - sort options.
-   * @param {number} lastActivity - number of days since last activity.
+   * @param {number} numberOfDaysSinceLastActivityStartDate - number of days since first activity.
+   * @param {number} numberOfDaysSinceLastActivityEndDate - number of days since last activity.
```

Comment:
Reminder to update the documentation to correspond with the new parameter names.

Reply:
done